src/Components/Files/FileUpload.tsx

```diff
@@ -0,0 +1,379 @@
+import { useState, ReactNode } from "react";
+import { FileUploadModel } from "../Patient/models.js";
+import Pagination from "../Common/Pagination.js";
+import { RESULTS_PER_PAGE_LIMIT } from "../../Common/constants.js";
+import { useTranslation } from "react-i18next";
+import ButtonV2 from "../Common/components/ButtonV2.js";
+import CareIcon, { IconName } from "../../CAREUI/icons/CareIcon.js";
+import TextFormField from "../Form/FormFields/TextFormField.js";
+import { NonReadOnlyUsers } from "../../Utils/AuthorizeFor.js";
+import AuthorizedChild from "../../CAREUI/misc/AuthorizedChild.js";
+import useAuthUser from "../../Common/hooks/useAuthUser.js";
+import useQuery from "../../Utils/request/useQuery.js";
+import routes from "../../Redux/api.js";
+import useFileUpload from "../../Utils/useFileUpload.js";
+import useFileManager from "../../Utils/useFileManager.js";
+import Tabs from "../Common/components/Tabs.js";
+import FileBlock from "./FileBlock.js";
+
+export const LinearProgressWithLabel = (props: any) => {
```

Comment:
Need types for the props
```suggestion
export const LinearProgressWithLabel = (props: {...}) => {
```